index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+   
+  </head>
+
+  <body>
+   <header id="title">Wangari Maathai</header>
+     
+   <main id="main">
+    <figure id="img-div">
+      <img id="image" src="https://i0.wp.com/thefootloosescribbler.com/wp-content/uploads/2018/09/wangari-2-e1537930415767.png?fit=600%2C653" alt="Woman activist" style="width:100%">
+      <figcaption id="img-caption">Wangari Maathai, Kenyan Activist</figcaption>
+    </figure>
```

Comment:
```suggestion
    <figure id="img-div">
      <img id="image" src="https://i0.wp.com/thefootloosescribbler.com/wp-content/uploads/2018/09/wangari-2-e1537930415767.png?fit=600%2C653" alt="Woman activist" style="width:100%">
      <figcaption id="img-caption">Wangari Maathai, Kenyan Activist</figcaption>
    </figure>
```
Try to use a class instead of an id in the "figure" tag and possibly remove the "IMG" id and directly style it in your CSS stylesheet in this case, it is recommended as you only have a single image to apply a class to multiple images of the same style or id to a unique style of an image.